Source/engine/animationinfo.cpp

```diff
@@ -27,9 +27,8 @@ int8_t AnimationInfo::getFrameToUseForRendering() const
 	if (currentFrame >= relevantFramesForDistributing_)
 		return currentFrame;
 
-	int16_t ticksSinceSequenceStarted = ticksSinceSequenceStarted_;
+	int16_t ticksSinceSequenceStarted = std::max<int16_t>(0, ticksSinceSequenceStarted_);

```

Comment:
but you've just made the condition below impossible to enter

Reply:
![image](https://github.com/user-attachments/assets/22e2edec-fd23-4e82-a8eb-f14a9883fead)